tools/trimmomatic.wdl

```diff
@@ -0,0 +1,72 @@
+version 1.1
+
+task trim {
+    meta {
+        description: "Performs paired end trimming of reads using the Trimmomatic tool."
+        outputs: {
+            trimmed_read_one: "The forward trimmed paired-end read in FASTQ format."
+            trimmed_read_two: "The reverse trimmed paired-end read in FASTQ format."
+        }
+    }
+
+    parameter_meta {
+        read_one: "Forward input FASTQ format file to run Trimmomatic on."
+        read_two: "Reverse input FASTQ format file to run Trimmomatic on."
+        output_name: "Output FASTQ format file of trimmed reads."
+        leading: "Cut bases off the start of a read, if below a threshold quality."
+        trailing: "Cut bases off the end of a read, if below a threshold quality."
+        window_size: "Specifies the number of bases to average across for the sliding window."
+        window_quality: "Specifies the average quality required in the sliding window."
+        minlen: "Drop the read if it is below a specified length."
+    }
+
+    input {
+        File read_one
+        File? read_two
+        String? illumina_clip
+        String prefix = sub(
+            basename(read_one),
+            "(([_.][rR](?:ead)?[12])((?:[_.-][^_.-]*?)*?))?\\.(fastq|fq)(\\.gz)?$",
+            ""  # Once replacing with capturing groups is supported, replace with group 3
+        )
+        String outfile_name_one = "~{prefix}.R1.trimmed.fastq.gz"
+        String outfile_name_two = "~{prefix}.R2.trimmed.fastq.gz"
```

Comment:
Move these to a private declaration (between the `input` block and the `command` block). Also rename them ot `String read_one_name` and `read_two_name`.